clients/geth/specular/rollup/services/validator/validator.go

```diff
@@ -26,9 +26,25 @@ type challengeCtx struct {
 	lastValidatedAssertion *rollupTypes.Assertion
 }
 
-var errAssertionOverflowedLocalInbox = fmt.Errorf("assertion overflowed inbox")
 var errValidationFailed = fmt.Errorf("validation failed")
 
+type errAssertionOverflowedLocalInbox struct {
+	Msg string
+}
+
+func (e *errAssertionOverflowedLocalInbox) Error() string {
+	return fmt.Sprint("assertion overflowed local inbox with msg:", e.Msg)
+}
+
+func (e *errAssertionOverflowedLocalInbox) Is(tgt error) bool {
+	_, ok := tgt.(*errAssertionOverflowedLocalInbox)
+	return ok
+}
```

Comment:
oh do you actually need to explicitly implement this? i figured it works automatically since it's a struct